_includes/shared/code/ccloud/config.properties

```diff
@@ -0,0 +1,15 @@
+# Required connection configs for Kafka producer, consumer, and admin
+bootstrap.servers={{ BOOTSTRAP SERVERS }}
+security.protocol=SASL_SSL
+sasl.jaas.config=org.apache.kafka.common.security.plain.PlainLoginModule   required username='{{ CLUSTER_API_KEY }}'   password='{{ CLUSTER_API_SECRET }}';
+sasl.mechanism=PLAIN
+# Required for correctness in Apache Kafka clients prior to 2.6
+client.dns.lookup=use_all_dns_ips
+
+# Best practice for Kafka producer to prevent data loss
+acks=all
+
+# Required connection configs for Confluent Cloud Schema Registry
+schema.registry.url=https://psrc-pgpdo.us-east-2.aws.confluent.cloud
```

Comment:
@awalther28 this endpoint `https://psrc-pgpdo.us-east-2.aws.confluent.cloud` needs to be obfuscated?

Reply:
ack updated in https://github.com/confluentinc/kafka-tutorials/pull/786/commits/9617b5570ca83d20d037f966c33e66b1fa0ab0a5